charts/fleet-crd/templates/gitjobs-crds.yaml

```diff
@@ -1,3 +1,3441 @@
+{{- if .Capabilities.APIVersions.Has "apiextensions.k8s.io/v1" -}}
```

Comment:
How did you generate this? Did you also update gitjob CI to include your change?

Reply:
Yes, the GitJob PR covers it: https://github.com/rancher/gitjob/pull/36

Oops, I forgot to add you for review there.

Reply:
Generated with the GitJob CRD gen.
```sh
( cd rancher-gitjob; go run pkg/crdgen/main.go > ../rancher-fleet/charts/fleet-crd/templates/gitjobs-crds.yaml )
```